histomicstk/cli/SuperpixelSegmentation/SuperpixelSegmentation.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<executable>
+  <category>HistomicsTK</category>
+  <title>Create Superpixel Pixelmap</title>
+  <description>Create a pixelmap image of superpixels using SLIC.</description>
+  <version>0.1.0</version>
+  <documentation-url>https://digitalslidearchive.github.io/HistomicsTK/</documentation-url>
+  <license>Apache 2.0</license>
+  <contributor>Kitware</contributor>
+  <acknowledgements>This work is part of the HistomicsTK project.</acknowledgements>
+  <parameters>
+    <label>IO</label>
+    <description>Input/output parameters.</description>
+    <image>
+        <name>inputImageFile</name>
+        <label>Input Image</label>
+        <description>Input image for superpixel segmentation</description>
+        <channel>input</channel>
+        <index>0</index>
+    </image>
+    <image fileExtensions=".tiff" reference="inputImageFile">
+      <name>outputImageFile</name>
+      <label>Output Image of Superpixel Pixelmap</label>
+      <description>Output Image of Superpixel Pixelmap (*.tiff)</description>
+      <channel>output</channel>
+      <index>1</index>
+    </image>
+    <file fileExtensions=".anot" reference="inputImageFile">
+      <name>outputAnnotationFile</name>
+      <label>Pixelmap Annotation</label>
+      <description>Annotation to display pixelmap on source (*.anot)</description>
+      <channel>output</channel>
+      <longflag>superpixel_annotation</longflag>
+    </file>
+    <region>
+      <name>roi</name>
+      <label>Analysis ROI</label>
+      <longflag>roi</longflag>
+      <description>Region of interest within which the analysis should be run</description>
+      <channel>input</channel>
+      <default>-1, -1, -1, -1</default>
+    </region>
+    <integer>
+       <name>tileSize</name>
+       <longflag>tileSize</longflag>
+       <label>Working tile size</label>
+       <description>Specify the size of the working tile. If there is no overlap, superpixxel boundaries will appear at these locations.</description>
+       <channel>input</channel>
+       <default>4096</default>
+    </integer>
+    <integer>
+       <name>superpixelSize</name>
+       <longflag>superpixelSize</longflag>
+       <label>Superpixel Size</label>
+       <description>Approximate width of the superpixel.</description>
+       <channel>input</channel>
+       <default>50</default>
+    </integer>
+    <boolean>
+      <name>overlap</name>
+      <label>Overlap</label>
+      <longflag>overlap</longflag>
+      <description>If specified, overlap tile computation to avoid edge effects.</description>
+      <channel>input</channel>
+      <default>false</default>
+    </boolean>
+    <boolean>
+      <name>boundaries</name>
+      <label>Boundaries</label>
+      <longflag>boundaries</longflag>
+      <description>If specified, draw separate superpixels as boundaries around actual superpixels.</description>
```

Comment:
The boundaries are separate pixelmap categories, but not independent superpixels.  This should be something like `Mark the boundary around each superpixel.  Boundaries are specified with a separate pixelmap index value that is always 1 higher than the pixelmap index value of the superpixel's interior`